6.2/test/kibana-6.1.bats

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bats
```

Comment:
(nit) presumably you meant to call this file `6.2/test/kibana-6.2.bats`, not `6.2/test/kibana-6.1.bats`.